joplin/css/pages/edit.scss

```diff
@@ -30,6 +30,26 @@
     display: none;
 }
 
+.coa-multiField-nopadding li {
+    padding: 0;
+}
+
+.coa-helpPanel fieldset {
+    padding: 0;
+}
+
+// tech debt: hiding help instead of removing, check scoping, bad idea?
```

Comment:
I'm not too worried about it, this seems like a good enough solution (unless we start running into weird visual bugs I haven't seen yet